SetCurrentVersion.cmd

```diff
@@ -1,7 +1,7 @@
 set MAJOR_PREVIOUS=1
 set MINOR_PREVIOUS=5
 set PATCH_PREVIOUS=12
-set PRERELEASE_PREVIOUS=-beta
+set PRERELEASE_PREVIOUS=-developer
```

Comment:
We should never release a -developer version. Did that happen? If not, this needs to remain 'beta'